.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: build
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+      - name: Build
+        run: fab build
+      - name: Check if worked
+        run: |
+          cd public
+          ls
```

Comment:
Does ls returns non-zero if empty dir?

Reply:
I would check the logs to see what ls would print, but I'll be using a test bucket